install.sh

```diff
@@ -129,6 +129,7 @@ then
   CHGRP=("/usr/bin/chgrp")
   GROUP="admin"
   TOUCH=("/usr/bin/touch")
+  INSTALL=("/usr/bin/install" -d -o "root" -g "wheel" -m "0750")
```

Comment:
```suggestion
  INSTALL=("/usr/bin/install" -d -o "root" -g "wheel" -m "0755")
```

Let's keep this world-readable.

Reply:
Fixed.